### PR TITLE
fix(shared-data): Fix the pipette schemas, tests

### PR DESCRIPTION
**Pull request by sfoster1**
The tests that were supposed to ajv the pipette definitions weren't set up properly, and the tests that were supposed to test that the tests were set up properly weren't set up properly, so we weren't testing the schemas so the schemas were wrong. Fix the schema-test setup, the test-setup-schema-test setup, and the schema.

### CODE REVIEW

**Review by ahiuchingau — APPROVED** [review]
Thank you!

**Review by ryanthecoder — APPROVED** [review]
lgtm

File: shared-data/python/tests/pipette/test_validate_schema.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ def test_check_all_models_are_valid() -> None:
         "ninety_six_channel": "96",
         "eight_channel": "multi",
     }
+    assert os.listdir(paths_to_validate), "You have a path wrong"
```
Comment (Laura-Danielle) [inline shared-data/python/tests/pipette/test_validate_schema.py:21]: Is this necessary?

**Review by Laura-Danielle — APPROVED** [review]
oopsie and lgtm

File: shared-data/python/tests/pipette/test_validate_schema.py (hunk 1 above)
Comment (sfoster1) [reply shared-data/python/tests/pipette/test_validate_schema.py:21]: IMO yes, we do it in the js tests too

**Pull request merged**